package.json

```diff
@@ -91,19 +98,25 @@
     "@types/passport-local": "^1.0.33",
     "@types/signale": "^1.2.0",
     "concurrently": "^3.6.1",
+    "connect-timeout": "^1.9.0",
```

Comment:
Which package is this a peer dependency for? I can't seem to find reference to it in your code.